package.json

```diff
@@ -55,7 +55,7 @@
     "mockgoose": "7.3.5",
     "mongodb": "^2.2.11",
     "mongodb-memory-server": "^1.6.1",
-    "mongoist": "1.4.3",
+    "mongoist": "2.3.0",
```

Comment:
Are we concerned with what necessitated the breaking change?

Reply:
Good point. It's the requirement to specify a port in the connection string: https://github.com/mongoist/mongoist/commit/90dd80bd1cfe812be0746ce99f6ec15a886f15d5

Note however that this is in `devDependencies`. The actual version of `mongoist` being used is specified in the dependent package -- `compose` in our case. We are moving from 2.0.0 to 2.3.0 there: https://github.com/mixmaxhq/compose/pull/8243/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

Reply:
Ah cool, didn't see that this was in `devDependencies` 👍.